install_config/oab_broker_configuration.adoc

```diff
@@ -553,6 +553,78 @@ is also required to configure the broker to use the Partner Registry URL:
     registry.connect.redhat.com
 ----
 
+[[oab-config-helm-chart-registry]]
+=== Helm Chart Registry
+
+Using the `helm` type allows you to consume Helm Charts from a Helm Chart Repository.
+
+[source,yaml]
+----
+registry:
+  - name: stable
+    type: helm
+    url: "https://kubernetes-charts.storage.googleapis.com"
+    runner: "docker.io/automationbroker/helm-runner:latest"
+    white_list:
+      - ".*"
+----
+
+*Note:* A heavy percentage of helm charts from the stable repository are not well suited for OpenShift.
+For more information see link:https://github.com/ansibleplaybookbundle/helm2bundle/wiki/Known-Issues[here].
+
+[[oab-config-api-v2-registry]]
+=== API V2 Docker Registry
+
+Using the `apiv2` type allows you to consume images from docker registries that implement the Docker Registry
```

Comment:
This sentence runs on. Can you reword "can be configured with the apiv2 registry adapter" into a second sentence?